package.json

```diff
@@ -1,17 +1,43 @@
 {
-  "name": "",
-  "version": "",
-  "description": "",
-  "author": "",
+  "name": "product-reviews-component",
+  "version": "1.0.0",
+  "description": "UI Clone of Amazon's product reviews section",
+  "author": "Victor Liu",
+  "private": true,
   "license": "",
   "engines": {
     "node": ">=6.13.0"
   },
+  "main": "main.js",
   "scripts": {
+    "develop": "webpack --mode development --watch",
+    "build": "webpack --mode production",
+    "seed": "node ./database/index.js"
   },
   "dependencies": {
+    "body-parser": "^1.18.3",
+    "dotenv": "^6.2.0",
+    "express": "^4.16.4",
+    "faker": "^4.1.0",
+    "lodash": "^4.17.11",
+    "mongodb": "^3.1.13",
+    "mongoose": "^5.4.10",
+    "react": "^16.8.1",
+    "react-dom": "^16.8.1",
+    "webpack-cli": "^3.2.3"
   },
   "devDependencies": {
-    "eslint-config-hackreactor": "git://github.com/reactorcore/eslint-config-hackreactor"
+    "@babel/core": "^7.2.2",
+    "@babel/preset-env": "^7.3.1",
+    "axios": "^0.18.0",
```

Comment:
axios should probably be a normal dependency not a dev dependency

Reply:
thanks I'll fix that now